clients/java/signalr/build.gradle

```diff
@@ -18,6 +18,7 @@ dependencies {
     testImplementation group: 'junit', name: 'junit', version: '4.12'
     implementation "org.java-websocket:Java-WebSocket:1.3.8"
     implementation 'com.google.code.gson:gson:2.8.5'
+    compile 'org.apache.httpcomponents:httpclient:4.5.6'
```

Comment:
New dependency for an implementation of an HttpClient

Reply:
This has been approved 👍

Reply:
Looks like [`compile` has been deprecated](https://docs.gradle.org/current/userguide/java_plugin.html#tab:configurations) and `implementation` should be used instead (like the other dependencies)